picard/tagger.py

```diff
@@ -451,7 +451,17 @@ def get_files_from_objects(self, objects, save=False):
     def save(self, objects):
         """Save the specified objects."""
         files = self.get_files_from_objects(objects, save=True)
-        for file in files:
+        coverOverwritePrompt = config.setting["cover_overwrite_prompt"]
+        for file, obj in zip(files, objects):
```

Comment:
I'm not 100% sure about this, but the lengths of `objects` and `files` are not always the same (a track can be associated with multiple files and an album/cluster with multiple tracks).